dockers/docker-teamd/config.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
>bash [](start = 7, length = 4)

I suggest don't limit to write bash script here. Name it as ./config, and provide shebang such as 'make -f' 'env python' ...
Then you can write this file wholly in python.